spp_change_request/views/change_request_view.xml

```diff
@@ -7,7 +7,7 @@ Part of OpenSPP. See LICENSE file for full copyright and licensing details.
         <field name="model">spp.change.request</field>
         <field name="priority">1</field>
         <field name="arch" type="xml">
-            <tree sample="1" decoration-danger="date_rejected != False">
+            <tree sample="1" decoration-danger="date_rejected != False" delete="0">
```

Comment:
I think this should be a context on action, we still can delete in form view if only have this change. Or it is me who not yet updated the code base

Reply:
There's no deletion in the form view. You can check in the code that the form already has delete="0".